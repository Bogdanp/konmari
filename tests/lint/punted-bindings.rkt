#lang racket/base

(provide
 (contract-out
  [a an-a?]))

(define a 'a)

(define (an-a? v)
  (eq? v 'a))

(lambda ()
  (show "hello")

  (let ()
    (define (show s)
      (displayln s))))
