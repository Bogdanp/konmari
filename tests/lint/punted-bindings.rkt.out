/punted-bindings.rkt:16:14:warning:identifier 'show' is never used
